### PR TITLE
Moved integration tests to integration test folder [Resolves #3432]

### LINKED ISSUE #3432: Move an integration test to integration test folder which is present in Unit test file [PART 5]
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

Currently some integration tests are present in unit test folder, so we can move them from this folder and add them to integration test folder.
https://github.com/publiclab/plots2/blob/faa03523b35ccaddd26f4bfc00a2c0bcce591b5b/test/unit/comment_test.rb#L260

### PR DESCRIPTION
**Pull request by oorjitchowdhary**
Fixes #3432 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by oorjitchowdhary** [comment]
@publiclab/reviewers pls review my PR and approve my task at GCI... Thanks :)

**Comment by oorjitchowdhary** [comment]
@namangupta01 @SidharthBansal @jywarren pls review... I can't claim any other tasks...

File: test/integration/token_comment_test.rb [hunk 1]
```diff
@@ -32,5 +32,21 @@ class TokenCommentTest < ActionDispatch::IntegrationTest
...
+    mail.subject = "Re: #{node.title} (##{node.nid})"
+    Comment.receive_mail(mail)
+    f = File.open('test/fixtures/incoming_test_emails/yahoo/final_parsed_comment.txt', 'r')
+    comment = Comment.last
+    user_email = mail.from.first
+    assert_equal comment.comment, f.read
+    assert_equal comment.nid, node.id
+    assert_equal comment.message_id, mail.message_id
+    assert_equal comment.comment_via, 1
+    assert_equal User.find(comment.uid).email, user_email
```
Comment (gauravano) [inline test/integration/token_comment_test.rb]: Move this test by creating a new file in `incoming_test_parsing_test` folder like done for Gmail - https://github.com/publiclab/plots2/tree/master/test/integration/incoming_mail_parsing_test

**Comment by oorjitchowdhary** [comment]
@gauravano Oh ok.. I will close #4001 ... I really didn't know 😞

File: test/integration/token_comment_test.rb (hunk 1 above)
Comment (oorjitchowdhary) [reply test/integration/token_comment_test.rb]: > Move this test by creating a new file in `incoming_test_parsing_test` folder like done for Gmail - https://github.com/publiclab/plots2/tree/master/test/integration/incoming_mail_parsing_test

So I have to remove the test from here or it is meant to be there for both locations?

**Comment by oorjitchowdhary** [comment]
I'll make a new PR with the requested changes

**Comment by jywarren** [comment]
It's ok, thank you for your interest and good energy!!!

**Pull request closed without merging**